File: sdkwork-app-sdk-typescript/src/types/exchange-rate-create-form.ts
/** 创建汇率表单 */
export interface ExchangeRateCreateForm {
  /** 基础货币代码 */
  baseCurrencyCode: string;
  /** 目标货币代码 */
  targetCurrencyCode: string;
  /** 汇率 */
  rate: number;
  /** 生效日期 */
  effectiveDate?: string;
}

File: sdkwork-app-sdk-java/src/main/java/com/sdkwork/app/model/ExchangeRateCreateForm.java
package com.sdkwork.app.model;

public class ExchangeRateCreateForm {
    private String baseCurrencyCode;
    private String targetCurrencyCode;
    private Double rate;
    private String effectiveDate;

    public String getBaseCurrencyCode() {
        return this.baseCurrencyCode;
    }
    
    public void setBaseCurrencyCode(String baseCurrencyCode) {
        this.baseCurrencyCode = baseCurrencyCode;
    }

    public String getTargetCurrencyCode() {
        return this.targetCurrencyCode;
    }
    
    public void setTargetCurrencyCode(String targetCurrencyCode) {
        this.targetCurrencyCode = targetCurrencyCode;
    }

    public Double getRate() {
        return this.rate;
    }
    
    public void setRate(Double rate) {
        this.rate = rate;
    }

    public String getEffectiveDate() {
        return this.effectiveDate;
    }
    
    public void setEffectiveDate(String effectiveDate) {
        this.effectiveDate = effectiveDate;
    }
}
